hack/dev-env/start-agent-managed.sh

```
@@ -24,7 +26,7 @@ SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
 export ARGOCD_AGENT_REMOTE_PORT=${ARGOCD_AGENT_REMOTE_PORT:-8443}
 
 if test "${REDIS_PASSWORD}" = ""; then
-    export REDIS_PASSWORD=$(kubectl get secret argocd-redis --context=vcluster-agent-managed -n argocd -o jsonpath='{.data.auth}' | base64 --decode)
+    export REDIS_PASSWORD=$(kubectl get secret argocd-redis --context=vcluster-agent-managed -n ${ARGOCD_MANAGED_NAMESPACE} -o jsonpath='{.data.auth}' | base64 --decode)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)